app/View/Components/SelectList.php

```diff
@@ -41,13 +41,17 @@
 
 class SelectList extends Component
 {
+    /**
+     * The options for the select list.
+     *
+     * @var mixed
+     */
```

Comment:
```suggestion
    /**
     * @var mixed
     */
```

And are you sure this should be mixed instead of an array? Can we add a type to the property? Where are we using this?

Reply:
@danharrin 

resolved first error.

Not getting you in your second comment.(And are you sure this should be mixed instead of an array? Can we add a type to the property? Where are we using this?)

I just checked and found that this component is not used anywhere. Also there is no view file available for this component which mention in render().

Reply:
@Orrison just checking, do you know of anywhere where we use this component either?

Reply:
@Orrison 
waiting for your input !

Reply:
I am not seeing, nor do I know of, any references/usages of this file. And like y'all said, the blade file is gone as well.